packages/pubsub/src/connection-pool.js

```diff
@@ -177,6 +190,23 @@ ConnectionPool.prototype.createConnection = function() {
     var connection = client.streamingPull(self.metadata_);
     var errorImmediateHandle;
 
+    if (self.isPaused) {
+      connection.pause();
+    }
+
+    self.once(CHANNEL_ERROR, function(err) {
```

Comment:
`err` isn't handled here

Reply:
It was a mistake to add that argument, we shouldn't be handling the error there.